src/classes/Scheduler.cls

```diff
@@ -0,0 +1,28 @@
+/*************************************************************************************************
+* This file is part of the Nebula Framework project, released under the MIT License.             *
+* See LICENSE file or go to https://github.com/jongpie/NebulaFramework for full license details. *
+*************************************************************************************************/
+public class Scheduler {
+    //CRON order - Seconds, Minutes, Hours, Day_of_month, Month, Day_of_week, Optional_year
+    public final static String DAILY_CRON = '0 {0} {1} * * ?';
+    public final static String HOURLY_CRON = '0 {0} * * * ?';
+    private final Schedulable scheduledClass;
+
+    public Scheduler (Schedulable scheduledClass)  {
+        this.scheduledClass = scheduledClass;
+    }
+
+    public String scheduleHourly(String jobName, String startingMinuteInHour) {
+        String hourlyCRON = String.format(HOURLY_CRON, new List<String>{startingMinuteInHour});
+        return this.schedule(jobName, hourlyCRON);
+    }
+
+    public String scheduleDaily(String jobName, List<String> startingHourAndMinute) {
```

Comment:
Just curious, what's the reason for using a list (that I assume contains 2 values, hour & minute), instead of 2 separate parameters (hour & minute)? Seems like this would require a bit more work to use since you'd always have to create a list and add those 2 values to the list.